src/main.jsx

```diff
@@ -1,6 +0,0 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import App from './examples/AddMore';
-import './css/style.css';
-
-ReactDOM.createRoot(document.getElementById('root')).render(<App />);
```

Comment:
Are you sure that you have to delete that file?

Reply:
> Are you sure that you have to delete that file? 

Yes this file is of no use